Packs/QualysVulnerabilityManagement/Integrations/QualysVulnerabilityManagement/QualysVulnerabilityManagement.py

```diff
@@ -8,297 +8,757 @@
+        'api_route': API_SUFFIX + '/scan/?action=launch',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-vm-scan-action': {
+        'api_route': API_SUFFIX + '/scan/',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-pc-scan-launch': {
+        'api_route': API_SUFFIX + '/scan/compliance/?action=launch',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-pc-scan-manage': {
+        'api_route': API_SUFFIX + '/scan/compliance/?action=launch',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-ip-add': {
+        'api_route': API_SUFFIX + '/asset/ip/?action=add',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-ip-update': {
+        'api_route': API_SUFFIX + '/asset/ip/?action=update',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-host-excluded-manage': {
+        'api_route': API_SUFFIX + '/asset/excluded_ip/',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-scheduled-report-launch': {
+        'api_route': API_SUFFIX + '/schedule/report/?action=launch_now',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-report-launch-map': {
+        'api_route': API_SUFFIX + '/report/?action=launch&report_type=Map',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-report-launch-scan-based-findings': {
+        'api_route': API_SUFFIX + '/report/?action=launch&report_type=Scan',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-report-launch-host-based-findings': {
+        'api_route': API_SUFFIX + '/report/?action=launch&report_type=Scan',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-report-launch-patch': {
+        'api_route': API_SUFFIX + '/report/?action=launch&report_type=Patch',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-report-launch-remediation': {
+        'api_route': API_SUFFIX + '/report/?action=launch&report_type=Remediation',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-report-launch-compliance': {
+        'api_route': API_SUFFIX + 'report/?action=launch&report_type=Compliance',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-report-launch-compliance-policy': {
+        'api_route': API_SUFFIX + '/report/?action=launch&report_type=Policy',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'qualys-virtual-host-manage': {
+        'api_route': API_SUFFIX + 'asset/vhost/',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+    'test-module': {
+        'api_route': API_SUFFIX + '/scan/?action=list',
+        'call_method': 'POST',
+        'resp_type': 'text',
+    },
+}
+
+commands_args_data: Dict[str, Any] = {
```

Comment:
Add a comment for this dictionary as you've done for the rest.

Reply:
fixed